font/src/ft/mod.rs

```diff
@@ -120,8 +124,8 @@ impl ::Rasterize for FreeTypeRasterizer {
 
         Ok(RasterizedGlyph {
             c: c,
-            top: glyph.bitmap_top(),
-            left: glyph.bitmap_left(),
+            top: glyph.bitmap_top() + self.glyph_offset_y,
```

Comment:
I don't think this is the right place to apply the offset. The `font` package should be very general. The offsets are very Alacritty-specific.

A better spot might be where the glyph cache calls into the rasterizer.